kernel/timer.c

```diff
@@ -411,7 +411,7 @@ tid_t tid_3;
 int timer_nr_tests = TIMER_TEST_SHOTS;
 
 #ifdef CONFIG_TIMER_TEST_SHUTDOWN
-#define timer_test_shutdown()		board_shutdown()
+#define timer_test_shutdown()		sim_shutdown()
```

Comment:
Could we adds board_shutdown() and invokes sim_shutdown() inside of it instead modifying this?
See, board_shutdown() is provided by core module and should have already been stubbed in include/target/arch.h.
While sim_shutdown() can only be found on riscv.